programs/ssl/ssl_server2.c

```diff
@@ -292,7 +294,7 @@ int main( void )
     "    renegotiation=%%d    default: 0 (disabled)\n"      \
     "    renegotiate=%%d      default: 0 (disabled)\n"      \
     "    renego_delay=%%d     default: -2 (library default)\n" \
-    "    renego_period=%%d    default: (library default)\n"
+    "    renego_period=%%d    default: (2^64 - 1)\n"
```

Comment:
I understand what you mean, so let me explain.

If not supplied in the command-line, the default value that will be stored in memory for `renego_period` IS `2^64 - 1` because of the change in line [118](https://github.com/ARMmbed/mbedtls/pull/729/files#diff-9803871cb5f1d60d6c3807c40b21ac8bR118). However, as you point out, only 48 bits of this value are used by the SSL module in the case for DTLS (this is what we are fixing with this PR), hence the "default" in that case is `2^48 -1` BUT still the value in memory for `renego_period` is `2^64 - 1` as it is 8 bytes.